src/main/java/net/datafaker/transformations/sql/SqlTransformer.java

```diff
@@ -219,7 +219,7 @@ private String handleObject(Object value) {
     }
 
     private String handledObjectToString(int length, String strValue) {
-        StringBuilder builder = new StringBuilder();
+        StringBuilder builder = new StringBuilder(32);
```

Comment:
I meant literally the length being passed in.

Reply:
done.